dmutils/status.py

```diff
@@ -26,6 +28,19 @@ def get_flags(current_app):
     return flags
 
 
+def get_disk_space_status(low_disk_threshold=5):
```

Comment:
Variable name needs an indication of units / scale.

Reply:
Updated.